ui/push-health/Health.jsx

```diff
@@ -40,12 +41,23 @@ export default class Health extends React.Component {
     this.setState({ user });
   };
 
-  updatePushHealth = () => {
-    this.setState({ healthData });
+  updatePushHealth = async () => {
```

Comment:
It might be helpful to utilize the [getData](https://github.com/mozilla/treeherder/blob/master/ui/helpers/http.js#L38) helper method (which uses async/await and checks for json and non-json errors like the 503s) in the `PushModel.getHealth`so you don't have to rewrite it here, you can just destructure the data (which could be data or error message) and failureStatus.

Reply:
Ahh, good call.  :)  I had forgotten about that. Thanks!

Reply:
@sarah-clements : Would you mind taking a look at my updates based your your comments?  I made a few changes to the shared component ``ErrorMessages`` that should be completely backward compatible.  But I'd value you double checking me on that.  :)